Assignment3.txt

```diff
@@ -0,0 +1,6 @@
+Amanda Cuevas
+1a) The complexity of the following code snippet is O(n) = n.
+1b) The complexity of the following code snippet is O(n) = n + n^2.
```

Comment:
Close! since it's big O notation, you don't have to care about the lower terms, so this just becomes O(n^2).